site/src/components/pages/hub/sandboxed-block-demo.tsx

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useEffect, useRef, VoidFunctionComponent } from "react";
+
+import { ExpandedBlockMetadata } from "../../../lib/blocks";
+
+export interface SandboxedBlockProps {
+  metadata: ExpandedBlockMetadata;
+  props: Record<string, unknown> | undefined;
+}
+
+export const SandboxedBlockDemo: VoidFunctionComponent<SandboxedBlockProps> = ({
+  metadata,
+  props,
+}) => {
+  const iframeRef = useRef<HTMLIFrameElement>(null);
+
+  const postBlockProps = useCallback(() => {
+    iframeRef.current?.contentWindow?.postMessage(JSON.stringify(props), "*");
+  }, [props]);
+
+  useEffect(() => {
+    postBlockProps();
+  }, [postBlockProps]);
+
+  const aliasBaseUrl = ""; // @todo
```

Comment:
@CiaranMn I guess I need some help here. It’s impossible to use `blocksandbox.org` because preview URLs are like `blockprotocol-xxx-hashintel.vercel.app`. I see no place to use the alias, so it’s a chicken and egg situation.